app/src/main/res/values/strings.xml

```diff
@@ -235,4 +235,5 @@
 
     <string name="delete_account_button_label">Delete Account</string>
     <string name="google_account_created_confirmation">Connected Google Account to your device successfully. Please wait while we retry the signup.</string>
+    <string name="generic_error_message_internet_connection">An error occurred. Check your internet connection</string>
```

Comment:
Change: Here, perhaps warn the user that the behavior is now undefined. The change may be saved or not, we can not provide a guarantee for either.

Reply:
Fixed